packages/tokens/src/global/borders.ts

```diff
@@ -0,0 +1,11 @@
+export interface BorderProps extends Array<number> {
```

Comment:
Optional, but i think the TS community prefers `<type>[]` [syntax](https://www.typescripttutorial.net/typescript-tutorial/typescript-array-type/#:~:text=In%20TypeScript%2C%20an%20array%20is,let%20arr%3A%20type%5B%5D%20syntax.), to avoid name conflict with [JS Array class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array).
```suggestion
export interface BorderProps extends number[] {
```

Reply:
I agree (it's also more consistent with what our IDEs show us). I think there are eslint rules to autofix this 🤔

Reply:
Initially I thought of doing it this way, but when I change it to `number[]` I get this error:
"An interface can only extend a qualified identifier/name with optional type arguments"

Googling I understood that this error is because we can only extend an identifier or a qualified name with optional type arguments when defining an interface, in this case `number[]` is an array of numbers, not an identifier or a qualified name and that's why the error happens.

So what I understand is that in this case I should use `Array<number>`, but it still makes me confused 🤔